src/VNNLIB/Syntax.agda

```diff
@@ -90,14 +91,23 @@ mutual
     [] : NetworkContext
     _∷_ : (Γ : NetworkContext) → NetworkDeclaration Γ → NetworkContext
 
+  ----------------------------------
+  -- Network congruence statement --
+  ----------------------------------
+  data NetworkEquivalence (Γ : NetworkContext) : Set where
+    equal-to : EqualNetworkVariable Γ → NetworkEquivalence Γ
```

Comment:
We also need to fill in the definition of `ValidIsomorphicToTarget` and `ValidEqualtoTarget` postulates. Which is probably just the definition of `NotEqualNetwork`, so maybe we should just move all three of those definitions to this file?